### PR TITLE
feat(spinbot): pull request metrics

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: spinbot/event/pull_request_closed_event_handler.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            log.warn('Unable to determine pull request for {}'.format(event))
+            return
+
+        merged = pull_request.merged
+        age = pull_request.closed_at - pull_request.created_at
+        change_size = pull_request.additions + pull_request.deletions
+        files_changed_count = pull_request.changed_files
+        comments = pull_request.comments
+
+        approved = next((r for r in pull_request.get_reviews() if r.state == 'APPROVED'), None) is not None
```
Comment (jtk54) [inline spinbot/event/pull_request_closed_event_handler.py]: Do we care about self-approvals here? I hope it wouldn't happen, but you never know.

Comment (lwander) [reply spinbot/event/pull_request_closed_event_handler.py]: They're no longer possible via github

Comment (lwander) [reply spinbot/event/pull_request_closed_event_handler.py]: image

**Comment by jtk54** [comment]
Nice.

**Pull request merged**